test/langtools/tools/javac/generics/inference/8015505/T8015505.java

```diff
@@ -2,7 +2,7 @@
  * @test /nodynamiccopyright/
  * @bug 8015505
  * @summary Spurious inference error when return type of generic method requires unchecked conversion to target
- * @compile/fail/ref=T8015505.out -Xlint:-options -source 7 -XDrawDiagnostics T8015505.java
+ * @compile/fail/ref=T8015505.out -Xlint:-options --release 7 -XDrawDiagnostics T8015505.java
```

Comment:
OK, I'll not bother with flagging more instances of `-options` ... `I suggest you use `grep` on the modified files in this PR.